ClientApp/src/components/pages/EmployeeDirectoryPage/EmployeeDirectoryPage.tsx

```diff
@@ -1,17 +1,91 @@
-import { PermissionsWrapper } from "components/reuseables";
-import React from "react";
-import { useRequireLogin } from "services";
-import { PermissionType } from "_generated";
+import {
+  AsyncComponent,
+  ErrorModal,
+  Header,
+  PermissionsWrapper,
+  useErrorHandling,
+} from "components";
+import _ from "lodash";
+import React, { useEffect, useState } from "react";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import { getAllEmployees, useRequireLogin } from "services";
+import { EmployeeOverview, PermissionType } from "_generated";
+import "./EmployeeDirectoryPage.scss";
+import { EmployeeDirectoryTable } from "./EmployeeDirectoryTable";
 
 export const EmployeeDirectoryPage: React.FC = (): JSX.Element => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [employees, setEmployees] = useState<EmployeeOverview[]>([]);
+  const { error, handleError } = useErrorHandling();
   useRequireLogin();
+
+  const sortAndGroup = (
+    emps: EmployeeOverview[],
+  ): _.Dictionary<EmployeeOverview[]> => {
+    return _(emps)
+      .sortBy(
+        (e: EmployeeOverview): string | null | undefined => e.manager?.fullName,
```

Comment:
To avoid nulls,
```suggestion
        (e: EmployeeOverview): string | undefined => e.manager?.fullName || undefined,
```
Similarly on line 32.